assets/app/views/browse/_pod-details.html

```diff
@@ -47,6 +47,9 @@
           <dt>State:</dt>
           <dd>
             <kubernetes-object-describe-container-state container-state="containerStatus.state"></kubernetes-object-describe-container-state>
+            <span ng-if="!containerStatus.state.running && !(pod | isDebugPod)">
```

Comment:
what if its running but never becoming ready? you may still want to debug because your readiness check isnt passing